wtf.go

```diff
@@ -104,51 +107,55 @@ func refreshAllWidgets() {
 	}
 }
 
+func watchForConfigChanges(app *tview.Application, configFlag *string) {
+	w := watcher.New()
+
+	// notify write events.
+	w.FilterOps(watcher.Write)
+
+	go func() {
+		for {
+			select {
+			case <-w.Event:
+				Pages.RemovePage("grid")
```

Comment:
No need for this line, if you add a page with the same identifier as an existing one, it over-writes the existing one: https://github.com/rivo/tview/blob/74643a2db581c4268b25079d7a3f7a9ed441d0e2/pages.go#L52

(I didn't know that until now. Good stuff.)